cpu.h

```diff
@@ -26,16 +25,10 @@ struct cpu
 	int snd;
 };
 
+
 extern struct cpu cpu;
 
-void cpu_reset();
-void div_advance(int cnt);
-void timer_advance(int cnt);
-void lcdc_advance(int cnt);
-void sound_advance(int cnt);
-void cpu_timers(int cnt);
-int cpu_emulate(int cycles);
+int return_globalpc();
```

Comment:
what's the point of removing these decls ?
it looks more and more as if you just copied the "community edition" gnuboy over this one.